ci/install-package.sh

```diff
@@ -160,29 +207,29 @@ then
     # This is needed before tue-get, so also new deps are installed.
     # After a tue-get run, we checkout forced, just to be sure.
 
-    # First install only the git repo of the package so that appropriate branch can be checked out later
-    echo -e "\e[35m\e[1mtue-get install ros-$PACKAGE --no-ros-deps\e[0m"
-    docker exec tue-env bash -c 'source ~/.bashrc; tue-get install ros-"$PACKAGE" --no-ros-deps'
-
     # Fetch the merged branch
-    echo -e "\e[35m\e[1mgit -C ~${TUE_SYSTEM_DIR#$DOCKER_HOME}/src/$PACKAGE fetch origin pull/$PULL_REQUEST/merge:PULLREQUEST\e[0m"
-    docker exec -t tue-env bash -c 'source ~/.bashrc; git -C "$TUE_SYSTEM_DIR"/src/"$PACKAGE" fetch origin pull/$PULL_REQUEST/merge:PULLREQUEST'
+    echo -e "\e[35m\e[1mgit -C ~${TUE_SYSTEM_DIR#$DOCKER_HOME}/src/$PACKAGE fetch origin $REF_NAME/$PULL_REQUEST/merge:PULLREQUEST\e[0m"
+    docker exec -t tue-env bash -c 'source ~/.bashrc; git -C "$TUE_SYSTEM_DIR"/src/"$PACKAGE" fetch origin "$REF_NAME"/"$PULL_REQUEST"/merge:PULLREQUEST'
 
     # Install the package completely
-    echo -e "\e[35m\e[1mtue-get install ros-$PACKAGE --test-depend --branch=PULLREQUEST\e[0m"
-    docker exec tue-env bash -c 'source ~/.bashrc; tue-get install ros-"$PACKAGE" --test-depend --branch=PULLREQUEST'
+    echo -e "\e[35m\e[1mtue-get install $PACKAGE --test-depend --branch=PULLREQUEST\e[0m"
+    docker exec tue-env bash -c 'source ~/.bashrc; tue-get install "$PACKAGE" --test-depend --branch=PULLREQUEST'
 
     # Checkout -f to be really sure
-    echo -e "\e[35m\e[1mgit -C ~${TUE_SYSTEM_DIR#$DOCKER_HOME}/src/$PACKAGE checkout -f PULLREQUEST --\e[0m"
-    docker exec -t tue-env bash -c 'source ~/.bashrc; git -C "$TUE_SYSTEM_DIR"/src/"$PACKAGE" checkout -f PULLREQUEST --'
+    echo -e "\e[35m\e[1mgit -C ~${TUE_SYSTEM_DIR#$DOCKER_HOME}/src/$PACKAGE checkout -f PULLREQUEST\e[0m"
+    docker exec -t tue-env bash -c 'source ~/.bashrc; git -C "$TUE_SYSTEM_DIR"/src/"$PACKAGE" checkout -f PULLREQUEST'
```

Comment:
These `--` are there to prevent git getting confused over something being a branch or a file/folder